### ISSUE TITLE
Use of headings for product name links

### ISSUE THREAD

**Issue by alisonrae** [PublicisSapient/accessible-ecommerce-demo#37]
Headings are meant to convey the document structure. When we make the product names headings, we end up with a really long list of headings for the page that people need to tab through. If someone is trying to navigate by headings, they are probably looking for a shortcut and don't want to tab through the whole document. We need a structure like:
H1 - Category name, e.g. Shirts
H2 - Filters
H3 - Filter type (e.g. Color)
H2 - Product Grid - Showing products X of X (part of the heading could be hidden)
--- here is where the products go, they should be links, not headings ---
H2 - Product Recommendations

Please remove heading tags from product name links.

**Comment by alisonrae** [comment]
Fixed
